assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js

```diff
@@ -17,22 +17,93 @@
 /**
  * WordPress dependencies
  */
+import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
 import Header from '../../../../components/Header';
 import HelpMenu from '../../../../components/help/HelpMenu';
+import Layout from '../../../../components/layout/Layout';
+import { Cell, Grid, Row } from '../../../../material-components';
+import PageHeader from '../../../../components/PageHeader';
+import AdBlockingSetupSVG from '../../../../../svg/graphics/ad-blocking-recovery-setup.svg';
+import Link from '../../../../components/Link';
+import {
+	BREAKPOINT_SMALL,
+	useBreakpoint,
+} from '../../../../hooks/useBreakpoint';
+import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+const { useSelect } = Data;
 
 export default function AdBlockingRecoveryApp() {
+	const breakpoint = useBreakpoint();
+
+	const settingsURL = useSelect(
+		( select ) =>
+			`${ select( CORE_SITE ).getAdminURL(
+				'googlesitekit-settings'
+			) }#/connected-services/adsense`
```

Comment:
This is a fancy way of getting the settings page URL... Perhaps we should create a separate selector that returns the URL for the settings page with different fragments? Something like `getSettingsPageURL(tab, target, action)`?

Reply:
Thanks, @tofumatt.